docs/source/en/model_doc/detr.mdx

```diff
@@ -112,6 +112,20 @@ Tips:
   [`~transformers.DetrFeatureExtractor.pad_and_create_pixel_mask`].
 - The size of the images will determine the amount of memory being used, and will thus determine the `batch_size`.
   It is advised to use a batch size of 2 per GPU. See [this Github thread](https://github.com/facebookresearch/detr/issues/150) for more info.
+- There are three ways to instantiate a DETR model (depending on what you prefer):
+
+          from transformers import DetrConfig
+
+      Option 1: instantiate DETR with pre-trained weights for entire model
+          model = DetrForObjectDetection.from_pretrained("facebook/resnet-50")
+
+      Option 2: instantiate DETR with randomly initialized weights for Transformer, but pre-trained weights for backbone
+          config = DetrConfig()
+          model = DetrForObjectDetection(config)
+
+      Option 3: instantiate DETR with randomly initialized weights for backbone + Transformer
+          config = DetrConfig(use_pretrained_backbone=False)
+          model = DetrForObjectDetection(config)
```

Comment:
```suggestion
      ```
      from transformers import DetrConfig

      Option 1: instantiate DETR with pre-trained weights for entire model
          model = DetrForObjectDetection.from_pretrained("facebook/resnet-50")

      Option 2: instantiate DETR with randomly initialized weights for Transformer, but pre-trained weights for backbone
          config = DetrConfig()
          model = DetrForObjectDetection(config)

      Option 3: instantiate DETR with randomly initialized weights for backbone + Transformer
          config = DetrConfig(use_pretrained_backbone=False)
          model = DetrForObjectDetection(config)
      ```
```
This won't properly render if you don't add the code statements.

Reply:
Oh I think you need to do ```python for the first statement

Reply:
Did exactly that https://github.com/huggingface/transformers/pull/17712/commits/66eae21bdf95db6edaf2274f3449a865c80552f8

Reply:
cc'ing @sgugger here